cmd/bee-split/main.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/file/splitter"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/spf13/cobra"
+)
+
+var (
+	outdir      string // flag variable, output dir for fsStore
+	inputLength int64  // flag variable, limit of data input
+	host        string // flag variable, http api host
+	port        int    // flag variable, http api port
+	noHttp      bool   // flag variable, skips http api if set
+	ssl         bool   // flag variable, uses https for api if set
+)
+
+// teeStore provides a storage.Putter that can put to multiple underlying storage.Putters
+type teeStore struct {
+	putters []storage.Putter
+}
+
+// newTeeStore creates a new teeStore
+func newTeeStore() *teeStore {
+	return &teeStore{}
+}
+
+// Add adds a storage.Putter
+func (t *teeStore) Add(putter storage.Putter) {
+	t.putters = append(t.putters, putter)
+}
+
+// Put implements storage.Putter
+func (t *teeStore) Put(ctx context.Context, mode storage.ModePut, chs ...swarm.Chunk) (exist []bool, err error) {
+	for _, putter := range t.putters {
+		_, err := putter.Put(ctx, mode, chs...)
+		if err != nil {
+			return []bool{}, err
```

Comment:
you can `return nil,err`, since `[]bool` is a slice, it is nullable, same goes to the other places in the PR where this is done